pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix

```diff
@@ -47,6 +30,26 @@ python3.pkgs.buildPythonApplication rec {
     ]
     # Inkstitch uses the builtin tomllib instead when Python >=3.11
     ++ lib.optional (pythonOlder "3.11") tomli;
+  pyEnv = python3.withPackages (_: dependencies);
+in
+python3.pkgs.buildPythonApplication {
+  pname = "inkstitch";
+  inherit version;
+  pyproject = false; # Uses a Makefile (yikes)
+
+  src = fetchFromGitHub {
+    owner = "inkstitch";
+    repo = "inkstitch";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    tags = "v${version}";
```

Reply:
Should be `tag`, singular, no?

Reply:
done in cd87669d6f040611e6e28bfd56c5d5772c40c560